artist/field/actuator_ideal.py

```diff
@@ -39,14 +43,18 @@ def motor_steps_to_angles(self, motor_steps: torch.Tensor) -> torch.Tensor:
         """
         return motor_steps
```

Comment:
```suggestion
        device = torch.device(device)
        return motor_steps
```
We do not really need this here as the base class implementation does not use device. However, it might help to not forget this line in the child classes.

Reply:
I dont understand why that would be necessary here.